README.md

```diff
@@ -21,17 +21,17 @@ tests:
 
 ### Run tests locally
 
-This program is distributed as a Docker image. To run a container locally:
+This program is distributed as a Docker image. To run a container locally (under a folder contains tests related yml file): 
 ```bash
 docker run --rm \
-    -v $(pwd)/tests.yaml:/tests/tests.yaml \
-    -e "TEST_HOST=example.com" \
+    -v $(pwd)/httpbin.yml:/tests/tests.yml \
+    -e "TEST_HOST=httpbin.org" \
     nytimes/httptest
 ```
 
 You should see an output similar to this:
 ```
-passed:  tests.yaml | root | /
+passed:  tests.yml | root | /
```

Comment:
I think this should be "httpbin.yml"

Reply:
actually it should be `tests.yml`. the volume mount map the host file to local file within container.